.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -379,6 +379,29 @@ jobs:
           cd ${ORCA_SUT_DIR}
           ./tests/ci/script.sh
 
+  drupal_next_major:
+    name: Drupal Next Major
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "8.1"
+          - "8.2"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 12.13.1
+        uses: actions/setup-node@v3
+        with:
+          node-version: 12.13.1
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          #coverage: xdebug
+      - name: Drupal Next Major
+        run: |
+          composer update drupal/core:^10.1 -W
+
```

Comment:
New job is required here, we can add "ISOLATED_TEST_ON_NEXT_MINOR" in orca-job matrix to existing jobs to validate D10.
```
matrix:
        orca-job:
          - ISOLATED_TEST_ON_CURRENT
          - ISOLATED_TEST_ON_NEXT_MINOR
```
ref - https://github.com/acquia/acquia_cms/pull/1561/files#diff-4eb666b493503cabe7a4f72501ebd718bc71bc8560928ffc0b9b9d031f15c8e1R130